File: Podcast_ObjC/Podcast_ObjC/Model/Podcast.m
//
//  Podcast.m
//  Podcasts_ObjC
//
//  Created by SEAN on 2018/10/30.
//  Copyright © 2018 SEAN. All rights reserved.
//

#import "Podcast.h"

@implementation Podcast
- (id)initWithDictionary: (nullable NSDictionary*)dict{
    self = [super init];
    if (self) {
        self.trackName = dict[@"trackName"];
        self.artistName = dict[@"artistName"];
        self.artworkUrl600 = dict[@"artworkUrl600"];
        self.trackCount = dict[@"trackCount"];
        self.feedUrl = dict[@"feedUrl"];
    }
    return self;
}

@end

File: MyPodcast/Model/Podcast.swift
//
//  Podcast.swift
//  MyPodcast
//
//  Created by SEAN on 2018/4/2.
//  Copyright © 2018年 SEAN. All rights reserved.
//

import UIKit

struct Podcast: Decodable {
    var trackName: String?
    var artistName: String?
    var artworkUrl600: String?
    var trackCount: Int?
    var feedUrl: String?
}
